app/src/main/java/com/misa/fresher/ui/sale/SaleFragment.kt

```diff
@@ -276,7 +277,7 @@ class SaleFragment : BaseFragment<FragmentSaleBinding>() {
         binding.btnCart.setOnClickListener {
             navigation.navigate(
                 R.id.action_fragment_sale_to_fragment_bill, bundleOf(
-                    "items" to selectedItems
+                    SELECTED_ITEMS to selectedItems
```

Comment:
naming convention
https://github.com/umesh0492/android-guidelines/blob/master/project_and_code_guidelines.md#2213-string-constants-naming-and-values